.ddev/config.yaml

```diff
@@ -0,0 +1,173 @@
+name: tea
+type: php
+docroot: ""
+php_version: "7.4"
+webserver_type: nginx-fpm
+router_http_port: "80"
+router_https_port: "443"
+xdebug_enabled: false
+additional_hostnames: []
+additional_fqdns: []
+mariadb_version: "10.2"
+mysql_version: ""
+provider: default
+use_dns_when_possible: true
+composer_version: ""
+hooks:
+  post-start:
+    - exec: "composer install --no-progress"
+
+# This config.yaml was created with ddev version v1.16.1
```

Comment:
The latest (stable) release is 1.16.7. Should we re-generate the file with the latest version in case there have been changes relevant to config file generation?